core/common/src/main/java/alluxio/wire/TieredIdentity.java

```diff
@@ -98,7 +132,9 @@ public String toString() {
      * @param tierName the name of the tier
      * @param value the value of the tier
```

Comment:
what do `null` values mean?

Reply:
they mean the tier value wasn't configured by the user